src/components/RichTextEditor/Plugins/BreakAfterPlugin/BreakAfterButton/BreakAfterToolbarButton.tsx

```diff
@@ -12,19 +14,26 @@ import {
 import React from 'react';
 import { KEY_ELEMENT_BREAK_AFTER } from '../BreakAfterPlugin';
 import { setBreakAfter } from '../utils/setBreakAfter';
+import { useRichTextEditorContext } from '@components/RichTextEditor/context/RichTextEditorContext';
 
 export const BreakAfterToolbarButton = ({ id, pluginKey = KEY_ELEMENT_BREAK_AFTER, ...props }: ToolbarButtonProps) => {
     const editor = usePlateEditorState(useEventPlateId(id));
     const isActive = !!editor?.selection && someNode(editor, { match: { breakAfterColumn: true } });
+    const { style } = useRichTextEditorContext();
+    const columnCount = Number(style?.columns);
 
     return (
         <ToolbarButton
             active={isActive}
-            tooltip={getTooltip('Add column break.\n[shift+ctrl+return]', 'bottom')}
-            onMouseDown={getPreventDefaultHandler(setBreakAfter, editor, {
-                value: !isActive,
-                key: pluginKey,
-            })}
+            tooltip={getTooltip('Break After Column.\n[shift+ctrl+return]')}
+            onMouseDown={(event) =>
+                isBreakAfterEnabled(editor, columnCount, isActive)
```

Comment:
Nice!!

Good idea. Implemented. I also adapted the tooltip.

Reply:
one more thing: we should make sure the the newly added element when hitting enter doesn't have a column-break, because you could add an arbitrary number of elements with a column-break that way

Reply:
oh, true! 😲

Reply:
@triggertoo this should now work. 

Also the ticket with the new backspace behaviour is added in here: https://app.clickup.com/t/86775uxuy

& The margin is removed: https://app.clickup.com/t/86775uqtm